docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
```

Comment:
Agreed with @harterrt that I'm still a little unsure of the top-level goals. Is the idea to get users to open more tabs? More tabs with different domains (ie, open facebook in firefox and not a neighboring chrome installation)? More accounts on the same domain (multiple google accounts)? To get users to feel "more secure" or more likely to use firefox when sharing a computer with multiple users?
Not all of these questions can/should be answered with the TxP study, but these are things that should be considered if usage is consistent/the UI is accessible and this is considered for a shield study in order to move it into release.